models/src/main/scala/coop/rchain/models/rholang/sorter/ExprSortMatcher.scala

```diff
@@ -220,6 +220,17 @@ private[sorter] object ExprSortMatcher extends Sortable[Expr] {
               )
             )
           )
+      case ETupleBody(tuple) =>
+        for {
+          sortedPars          <- tuple.ps.toList.traverse(Sortable[Par].sortMatch[F])
+          connectiveUsedScore = if (tuple.connectiveUsed) 1 else 0
```

Comment:
Nice! Did the tests enforce the `connectiveUsed` inclusion as well?

Reply:
Yes. You can see the failing test above.